tark/tark_web/urls.py

```diff
@@ -136,6 +136,20 @@
         )
     ),
 
+    # new mane list page
+    url(
+        r'^mane_list_new/$',
+        mane_list_new,
+        name="mane_list_new"
+    ),
+   
+   # new mane GRCh37 page
+    url(
+        r'^mane_GRCh37_list/$',
```

Comment:
Loading this page is is quite slow, it took ~50 seconds to load locally and sometimes I get `Oops, something went wrong` when I try to access it on [dev-tark](https://dev-tark.ensembl.org/web/mane_GRCh37_list/), probably due to a timeout error.
Is there a way to improve the response time?

Reply:
Apologies @bilalebi . I have now included pagination on the query set. This should load the pages faster.

Reply:
Thank @ranjit-s, it's much quicker now 👍